src/LEGO.AsyncAPI.Bindings/Sqs/SqsChannelBinding.cs

```diff
@@ -0,0 +1,81 @@
+namespace LEGO.AsyncAPI.Bindings.Sqs
+{
+    using System;
+    using LEGO.AsyncAPI.Readers.ParseNodes;
+    using LEGO.AsyncAPI.Writers;
+
+    /// <summary>
+    /// This object contains information about the channel representation in SQS.
+    /// </summary>
+    public class SqsChannelBinding : ChannelBinding<SqsChannelBinding>
+    {
+        /// <summary>
+        /// A definition of the queue that will be used as the channel.
+        /// </summary>
+        public Queue Queue { get; set; }
+
+        /// <summary>
+        /// A definition of the queue that will be used for un-processable messages.
+        /// </summary>
+        public Queue DeadLetterQueue { get; set; }
+
+        public override string BindingKey => "sqs";
+
+        protected override FixedFieldMap<SqsChannelBinding> FixedFieldMap => new()
+        {
+            { "queue", (a, n) => { a.Queue = n.ParseMapWithExtensions(this.queueFixedFields); } },
+            { "deadLetterQueue", (a, n) => { a.DeadLetterQueue = n.ParseMapWithExtensions(this.queueFixedFields); } },
+        };
+
+        private FixedFieldMap<Queue> queueFixedFields => new()
+        {
+            { "name", (a, n) => { a.Name = n.GetScalarValue(); } },
```

Comment:
lookjs like $Ref is missing

Reply:
But I dont quite understand the $Ref in the spec.
> Allows for an external definition of a queue. The referenced structure MUST be in the format of a Queue. If there are conflicts between the referenced definition and this Queue's definition, the behavior is undefined.

Defined where?

Reply:
Ive raised the same question in the Binding PR

Reply:
I think this is intended to allow you to define your queue specification as an external file or remotely as an endpoint. So it would still just be AsyncAPI in the same way that message and schema definitions can be `$ref`. For `$ref` to external files do they generally get explicitly defined in the object model like we have on the `Queue` object? I'm not sure how we unit test it.

Reply:
Thinking about it I am little doubtful on the value of having $ref just for this queue property so I might push for its removal.

Reply:
We dont have support for external specs currently though.